docs/FAQ.md

```diff
@@ -0,0 +1,67 @@
+# Frequently Asked Questions
+
+Links to sections:
+- [Common](#common)
+- [PyTorch](#pytorch)
+- [TensorFlow](#tensorflow)
+- [ONNX](#onnx)
+
+
+## Common
+
+### Does the Neural Network *Compression* Framework provide *lossless compression*?
+Not in the way the term "lossless compression" usually appears in literature. Under "compression" we mean the preparation of the model for *future* efficient execution of this model in the OpenVINO Inference Engine. Under "future" we mean that the process of compression is usually an offline, one-time step before the model is being used in production, which provides a new model object that could then be used instead of the original to run faster and take up lower memory without significantly losing accuracy.
+
+No *compression* in the sense of archiving or entropy coding is being done during NNCF compression.
+
+### I don't see any improvements after applying the `*_sparsity` algorithms
+The sparsity algorithms introduce unstructured sparsity which can only be taken advantage of in terms of performance by using specialized hardware and/or software runtimes. Within the scope of these algorithms, NNCF provides functionally correct models with non-salient weights simply zeroed out, which does not lead to the reduction of the model checkpoint size. The models can, however, be used for benchmarking experimental/future hardware or runtimes, and for SOTA claims of applying unstructured sparsity on a given model architecture.
+
+For an opportunity to observably increase performance by omitting unnecessary computations in the model, consider using the [filter pruning](./compression_algorithms/Pruning.md) algorithm. Models compressed with this algorithm can be executed more efficiently within OpenVINO Inference Engine runtime when compared to the uncompressed counterparts.
+
+### Can I use NNCF-compressed models with runtimes other than OpenVINO Inference Engine?
+While this is certainly possible in some cases, with a beneficial outcome even, we recommend NNCF as a way to get the most out of your setup based on OpenVINO Inference Engine inference.
```

Comment:
I think we should clearly state about the ONNX Execution Provider.

Reply:
Done